mipengine/common/node_tasks_DTOs.py

```diff
@@ -13,19 +13,6 @@ class ColumnInfo:
     name: str
     data_type: str
 
-    def __post_init__(self):
-        allowed_types = {"int", "text", "real"}
-        self.data_type = str.lower(self.data_type)
-        if self.data_type not in allowed_types:
-            raise TypeError(
-                f"Column can have one of the following types: {allowed_types}"
```

Comment:
This should not be removed completely.

It's one thing that we shouldn't test for sql injection and another thing that we shouldn't validate anything.

The allowed data types is still a "thing" and it should remain there.